src/explore-education-statistics-admin/src/hooks/useReleaseServiceStatus.ts

```diff
@@ -0,0 +1,84 @@
+import { StatusBlockProps } from '@admin/components/StatusBlock';
+import { getStatusDetail } from '@admin/components/ReleaseServiceStatus';
+import releaseService, {
+  ReleaseStageStatuses,
+} from '@admin/services/releaseService';
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { forceCheck } from 'react-lazyload';
+
+export interface StatusDetail {
+  color: StatusBlockProps['color'];
+  text: string;
+}
+
+interface Props {
```

Comment:
Not a biggy, but this isn't a component so would suggest naming this 'options' instead of 'props' (same applies for for any util functions you might write in the future as well)